Library/Formula/g3log.rb

```diff
@@ -0,0 +1,34 @@
+class G3log < Formula
+  desc "asynchronous, \"crash safe\", logger that is easy to use."
```

Comment:
Nope, the rest of the formula is ok :smiley: 

Alternatively, you could use single *inner* quotes to both keep the double (outer) quotes and avoid escaping.